<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MultiSigAddressDialog</class>
 <widget class="QDialog" name="MultiSigAddressDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>534</width>
    <height>164</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Add Multisignature Address</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3">
   <item>
    <widget class="QStackedWidget" name="stackedWidget">
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="mnAccept">
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="0" column="0">
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <layout class="QHBoxLayout" name="enterMLayout">
           <item>
            <widget class="QSpinBox" name="enterMSpinbox">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="minimum">
              <number>1</number>
             </property>
             <property name="maximum">
              <number>16</number>
             </property>
             <property name="value">
              <number>1</number>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="enterMLabel">
             <property name="text">
              <string>Enter the minimum number of signatures required to sign transactions</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="enterNLayout">
           <item>
            <widget class="QSpinBox" name="enterNSpinbox">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="minimum">
              <number>2</number>
             </property>
             <property name="maximum">
              <number>16</number>
             </property>
             <property name="value">
              <number>2</number>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="enterNLabel">
             <property name="text">
              <string>Enter the total number of addresses to validate the signatures</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="continueButtonLayout">
           <item>
            <widget class="QPushButton" name="continueButton">
             <property name="toolTip">
              <string>Sign the message to prove you own this PIVX address</string>
             </property>
             <property name="text">
              <string>&amp;Continue</string>
             </property>
             <property name="icon">
              <iconset resource="../pivx.qrc">
               <normaloff>:/icons/continue</normaloff>:/icons/continue</iconset>
             </property>
             <property name="autoDefault">
              <bool>false</bool>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="continueSpacer_1">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>48</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <widget class="QLabel" name="continueStatus">
             <property name="font">
              <font>
               <weight>75</weight>
               <bold>true</bold>
              </font>
             </property>
             <property name="text">
              <string/>
             </property>
             <property name="wordWrap">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="continueSpacer_2">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>48</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="addressAccept">
      <layout class="QGridLayout" name="gridLayout_3">
       <item row="0" column="0">
        <layout class="QVBoxLayout" name="addressLayout">
         <item>
          <widget class="QScrollArea" name="addressInputScrollArea">
           <property name="widgetResizable">
            <bool>true</bool>
           </property>
           <widget class="QWidget" name="addressList">
            <property name="geometry">
             <rect>
              <x>0</x>
              <y>0</y>
              <width>98</width>
              <height>28</height>
             </rect>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout_4"/>
           </widget>
          </widget>
         </item>
         <item>
          <layout class="QHBoxLayout" name="exitDialog">
           <item>
            <widget class="QPushButton" name="createAddressButton">
             <property name="toolTip">
              <string>Sign the message to prove you own this PIVX address</string>
             </property>
             <property name="text">
              <string>&amp;Create Address</string>
             </property>
             <property name="icon">
              <iconset resource="../pivx.qrc">
               <normaloff>:/icons/edit</normaloff>:/icons/edit</iconset>
             </property>
             <property name="autoDefault">
              <bool>false</bool>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="clearButton">
             <property name="toolTip">
              <string>Reset all sign message fields</string>
             </property>
             <property name="text">
              <string>Clear &amp;All</string>
             </property>
             <property name="icon">
              <iconset resource="../pivx.qrc">
               <normaloff>:/icons/remove</normaloff>:/icons/remove</iconset>
             </property>
             <property name="autoDefault">
              <bool>false</bool>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="exitSpacer_1">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>48</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <widget class="QLabel" name="exitStatus">
             <property name="font">
              <font>
               <weight>75</weight>
               <bold>true</bold>
              </font>
             </property>
             <property name="text">
              <string/>
             </property>
             <property name="wordWrap">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="exitSpacer_2">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>48</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../pivx.qrc"/>
 </resources>
 <connections/>
</ui>
